forge16/src/main/java/com/envyful/poke/removal/forge/command/PokeRemovalCommand.java

```diff
@@ -18,7 +31,96 @@ public class PokeRemovalCommand {
+
+                sender.sendMessage(new StringTextComponent(TextFormatting.GOLD + "Specs: "), Util.NIL_UUID);
+                int count = 0;
+                for (ITextComponent component: components) {
+                    count++;
+                    sender.sendMessage(component, Util.NIL_UUID);
+                    if (count > 40)
+                    {
+                        String fileName = "PokeRemoval/spec-" + new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss").format(new Date()) + ".txt";
+                        File specFile = new File(FMLPaths.CONFIGDIR.get().toFile(), fileName);
+                        try {
+                            Files.write(specFile.toPath(), getSpecListStr(search, Math.max(size, 4)), StandardCharsets.UTF_8);
+                            sender.sendMessage(new StringTextComponent(TextFormatting.GOLD + "List was too big full list is saved in: " + fileName), Util.NIL_UUID);
+                        } catch (IOException e) {
+                            sender.sendMessage(new StringTextComponent(TextFormatting.RED + "List was too big full list didn't save due to an error: " + fileName), Util.NIL_UUID);
+                            e.printStackTrace();
+                        }
+
+                        break;
+                    }
+                }
+            };
+
+            Thread thread = new Thread(specsTask);
+            thread.setName("SpecsThread");
+            thread.start();
+
+            return;
+        }
+
+        sender.sendMessage(new StringTextComponent("/pokeremoval reload").withStyle(TextFormatting.RED), Util.NIL_UUID);
+        sender.sendMessage(new StringTextComponent("/pokeremoval specs <items per row> <search>").withStyle(TextFormatting.RED), Util.NIL_UUID);
+    }
+
+    private List<ITextComponent> getSpecListTC(String search, int size){
+        final List<ITextComponent> components = new ArrayList<>();
+
+        for(String spec : getSpecListStr(search, size)) {
+            components.add(new StringTextComponent(TextFormatting.DARK_AQUA + spec.replace(", ",TextFormatting.GOLD + ", " + TextFormatting.DARK_AQUA)));
+        }
+
+        return components;
+    }
+
+    private List<String> getSpecListStr(String search, int size){
+        final AtomicInteger atomicInteger = new AtomicInteger(0);
+
+        Collection<List<String>> data;
+
+        if(search != null && !search.isEmpty()){
+            data = PixelmonCommandUtils.SPEC_REQUIREMENTS.stream().filter(s ->  s.contains(search)).collect(Collectors.groupingBy(it -> atomicInteger.getAndIncrement() / size)).values();
+        }
+        else {
+            data =PixelmonCommandUtils.SPEC_REQUIREMENTS.stream().collect(Collectors.groupingBy(str -> atomicInteger.getAndIncrement() / size)).values();
+        }
+
+        return data.stream().map(row -> String.join(", ", row)).collect(Collectors.toList());
+    }
+
+    private void Reload(ICommandSource sender){
```

Comment:
There's no reason for this to be it's own method; but additionally, it should be named following proper Java conventions